meshtastic/tests/test_smoke1.py

```
@@ -1,119 +1,171 @@
+        if result == 0:
+            return result, output
+        if attempt + 1 < RESTORE_ATTEMPTS:
+            time.sleep(RESTORE_RETRY_DELAY_SECONDS)
+    return result, output
+
+
+@pytest.fixture(scope="module", autouse=True)
+def restore_smoke1_module_config() -> None:
+    """Export baseline config before smoke1 and restore it after module completion."""
+    backup_path = Path(
+        f"/tmp/meshtastic-smoke1-module-backup-{int(time.time())}.yaml"
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/meshtastic-smoke1-module-backup-"

(S108)